projects/rtmpose/examples/onnxruntime/main.py

```diff
@@ -0,0 +1,472 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import argparse
+import time
+from typing import List, Tuple
+
+import cv2
+import loguru
```

Comment:
I assume that `loguru` is not installed by users. How about add a guide like `pip install loguru` in README.md as a kind reminder? I think it would be helpful for those new learners :)

Reply:
Thanks for your kind advice, I will update my tutorial.